Project.toml

```diff
@@ -1,10 +1,11 @@
 name = "Integrals"
 uuid = "de52edbc-65ea-441a-8357-d3a637375a31"
 authors = ["Chris Rackauckas <accounts@chrisrackauckas.com>"]
-version = "3.6.0"
+version = "3.7.0"
 
 [deps]
 CommonSolve = "38540f10-b2f7-11e9-35d8-d573e4eb0ff2"
+FastGaussQuadrature = "442a2c76-b920-505d-bb47-c5924d526838"
```

Comment:
I think I've now addressed this. I've never worked with extensions before so maybe I've done some things in a weird way..

I commented out Lines 3-21 of the new test file gaussian_quadrature_tests.jl since I couldn't see how to get the functions gauss_legendre and composite_gauss_legendre out of the extension file. They are covered by Lines 23 onward though, so maybe it's OK?